.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  checks:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 11.0.11
```

Comment:
Why not just run on the adoptopenjdk image in docker instead of manually installing it?
See reference: https://github.com/FAForever/faf-java-api/blob/develop/.github/workflows/build.yaml#L9

Reply:
didnt know it was thinking

Reply:
Does that image have javafx bundled as well?

Reply:
JavaFX is a regular dependency. It's not bundled since JDK8

Reply:
Yes I was just wondering if this image included javafx if not that might be a reason to just build it as that is just more straight forward.

Reply:
Straight forward is to use the same ci pipeline for all java builds (or at least as close as possible).
Straight forward is to use a docker image as that is decoupled from the underlying OS. Github just changed it from ubuntu 18.04 to 20.04 and that ofc broke stuff. Doesn't happen with docker images.